aip/general/0123.md

```diff
@@ -110,6 +114,7 @@ such as UpperCamelCase and snake_case.
 
 <!-- prettier-ignore-start -->
 [aip-122]: ./0122.md
+[aip-4231]: ./client-libraries/4231.md
```

Comment:
Note: this link will fail, like all the other client library links at the moment. Noah is looking into it.